orangecontrib/text/widgets/owconcordance.py

```diff
@@ -85,7 +85,7 @@ def flags(self, _):
         return Qt.ItemIsEnabled | Qt.ItemIsSelectable
 
     def rowCount(self, parent=None, *args, **kwargs):
-        return 0 if parent.isValid() or self.word_index is None \
+        return 0 if parent and parent.isValid() or self.word_index is None \
```

Comment:
This still fails if I select some documents in Concordance and then remove the data. It is fixed in #232.